DocAggregatorDotnetCore/TomAggregator.cs

```diff
@@ -38,15 +33,22 @@ public void DoSomeWork()
 
             Directory.CreateDirectory(outputDirectoryPath);
 
-            //Console.WriteLine("New folders to go in: " + outputDirectoryPath);
+            //make array of property refs
+            var ctx = new RiggsTestContext();
```

Comment:
You should ideally use a `using` statement when creating an Entity Framework context, like so:

using(var ctx = new RiggsTestContext()) {
  // CODE HERE
}

This ensures all the contexts resources are cleaned up straight away, and it's nice to consider that single block of DB work as a transaction.